### PR TITLE
Rentbw modeling systemv

### PR DESCRIPTION
**Pull request by deckb**
## Change Description

Use ISO compliant timestamp strings for input and config

## Deployment Changes
- [ ] Deployment Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Max-B1) [inline .vscode/settings.json:1]: Does the project need this file?

File: _model_config.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "net": {
+        "assumed_stake_weight": 944076307,
+        "current_weight_ratio": 1000000000000000,
+        "decay_secs": 86400,
+        "exponent": 4,
+        "max_price": "10000000.0000 TST",
+        "min_price": "0.0000 TST",
+        "target_timestamp": "2022-01-01T00:00:00.000",
```
Comment (Max-B1) [inline _model_config.json:9]: There is no timezone in the timestamp

File: _rentbw_input.csv [hunk 3]
```diff
@@ -0,0 +1,102 @@
+datetime,function,payer,receiver,days,net_frac,cpu_frac,max_payment,queue_max
+2021-01-01T19:00:00,configrentbw,test.json,0,0,0,0,0,0
```
Comment (Max-B1) [inline _rentbw_input.csv:2]: There is no timezone in the timestamp

**Review by Max-B1 — CHANGES_REQUESTED** [review]
There is no timezone in timestamps. I suggest to use ISO-8601 and provide all timestamps in UTC. Example: 2020-09-18T07:58:07.000Z

File: .vscode/settings.json (hunk 1 above)
Comment (deckb) [reply .vscode/settings.json]: It does not. I removed it and added to .gitignore

File: _model_config.json (hunk 2 above)
Comment (deckb) [reply _model_config.json:9]: I attempted to add the "Z" to the end and got the message "unable to convert ISO-formatted string to fc::time_point_sec" from 
https://github.com/EOSIO/eos/blob/0d87dff8bee56179aa01472dd00a089b2aa7b9fa/libraries/fc/src/time.cpp#L44

File: _rentbw_input.csv (hunk 3 above)
Comment (deckb) [reply _rentbw_input.csv:2]: same as above.

**Review by Max-B1 — APPROVED** [review]
I confirm that fc::time_point can't parse time zones when converting strings to time_point.
I'll work on parsing time zones later.

**Pull request merged**